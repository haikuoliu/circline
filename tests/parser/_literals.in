20;
20.0;
"str";
