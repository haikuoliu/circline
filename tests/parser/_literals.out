Expr(Num_Lit(20));
Expr(Num_Lit(20.));
Expr(String_Lit(str));
